elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -132,7 +133,8 @@ def render(element_html, data):
         html_params = {
             'submission': True,
             'parse_error': parse_error,
-            'uuid': pl.get_uuid()
+            'uuid': pl.get_uuid(),
+            'hide_letter_keys': pl.get_boolean_attrib(element, 'hide-letter-keys', HIDE_LETTER_KEYS_DEFAULT)
```

Comment:
Lets add this to the `html_params` in the question panel as well.  And conditionally hide/show the key in the answer panel.

Reply:
Should be fixed in 7920de8c33b57643dcb2e83a069d6213d5ec5923